src/components/ucp-tabs/overview/recent-folders.tsx

```diff
@@ -59,12 +56,16 @@ export default function RecentFolders() {
 
   // needs better loading site
   if (recentFolderResult.isEmpty()) {
-    return <p>{t('gui-general:loading')}</p>;
+    return (
+      <p>
+        <Message message="loading" />
+      </p>
+    );
   }
 
   return (
     <div className="text-input">
-      <label htmlFor="browseresult">{t('gui-landing:select.folder')}</label>
```

Comment:
Why is the label removed?

Reply:
Could not get it to be accepted by ESLint. Just decided to flat out replace it with a normal text.  
Also, clicking a label also triggered the input... which I found redundant, cosidering the big button beneath.